package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "sprint_11",
+  "version": "0.0.1",
+  "description": "11th sprint. making project via webpack",
+  "main": "src/\u001b[B\u001b[B\u001b[B\u001b[B\u001b[B\u001b[B\u001b[D\u001b[D\u001b[D\u001b[D./sr\u001b[2~c/index.js\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001bcript/index.js",
```

Можно лучше: исправить путь до скрипта в секции `main` в `package.json`.
Путь в секции `main` используется, когда NPM-пакет подключается через `import` или `require`. В текущем пути какие-то непонятные символы в unicode, нужно от них избавиться.
```suggestion
  "main": "src/index.js",
```